docs/source/1.0/spec/core/auth-traits.rst

```diff
@@ -305,76 +305,148 @@ Value type
     shape.
 
 Operations that are not annotated with the ``auth`` trait inherit the ``auth``
-trait of the service they are bound to, and if the operation is not annotated
-with the ``auth`` trait, then the operation is expected to support each of
-the :ref:`authentication scheme traits <authDefinition-trait>` applied to the
+trait of the service they are bound to. If the operation is not annotated with
+the ``auth`` trait, and the service it is bound to is also not annotated with
+the ``auth`` trait, then the operation is expected to support each of the
+:ref:`authentication scheme traits <authDefinition-trait>` applied to the
 service. Each entry in the ``auth`` trait is a shape ID that MUST refer to an
 authentication scheme trait applied to the service in which it is bound.
 
-The following example defines two operations:
+The following example defines all combinations in which ``auth`` can be applied
+to services and operations:
 
-* OperationA defines an explicit list of the authentication schemes it
-  supports using the ``auth`` trait.
-* OperationB is not annotated with the ``auth`` trait, so the schemes
-  supported by this operation inherit all of the authentication schemes
-  applied to the service.
+* ``ServiceWithNoAuthTrait`` does not use the ``auth`` trait and binds two
+  operations:
+
+  * ``OperationA`` is not annoated with the ``auth`` trait and inherits all
+    of the authentication scheme applied to the service.
+
+  * ``OperationB`` is annoated with the ``auth`` defines an explicit list of
```

Comment:
```suggestion
  * ``OperationB`` is annoated with the ``auth`` defines and explicit list of
```

Is that what you meant?

Reply:
This also has the `annoated` -> `annotated` typo.

Reply:
I've made the relevant changes. Thanks